CHANGELOG.md

```diff
@@ -44,16 +44,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added error constants for matching against both specific and generic unique constraint errors raised by the underlying database driver. (thanks @mbezhanov)
+- Added support for regular expressions in the `only` and `except` table filters. (thanks @mbezhanov)
 
 ### Removed
 
 - Remove MS SQL artifacts. (thanks @mbezhanov)
 
 ### Fixed
 
-- Removed unnecessary import of `strings` in `bobfactory_random.go`.
-- Fixed data races in unit tests.
+- Removed unnecessary import of `strings` in `bobfactory_random.go`. (thanks @mbezhanov)
```

Comment:
```suggestion
- Removed unnecessary import of `strings` in `bobfactory_random.go`. (thanks @mbezhanov)
- Removed unnecessary import of `strings` in `bobfactory_random.go`.
```

Reply:
Whoops, so sorry about this, I messed it up in the hurry.

I'll fix and rebase tomorrow morning, when I get back to my workstation 🙂

Reply:
Fixed :heavy_check_mark: